lighthouse-core/lib/page-functions.js

```diff
@@ -153,30 +153,74 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
 
 /**
  * Computes a memory/CPU performance benchmark index to determine rough device class.
+ * @see https://github.com/GoogleChrome/lighthouse/issues/9085
  * @see https://docs.google.com/spreadsheets/d/1E0gZwKsxegudkjJl8Fki_sOwHKpqgXwt8aBAfuUaB8A/edit?usp=sharing
  *
- * The benchmark creates a string of length 100,000 in a loop.
- * The returned index is the number of times per second the string can be created.
+ * Historically, this benchmark created a string of length 100,000 in a loop, and returned
```

Comment:
Can you add the LH version for the change? ie `Historically (until LH 6.3), ...`:

Reply:
```suggestion
 * Historically (until LH 6.3), this benchmark created a string of length 100,000 in a loop, and returned
```